CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
Adding this behavior into the main project would just need to add the references to these new dlls to the Nuget package, what else should be needed? 

I am not understanding your concern with deployment, maybe I am not experienced enough on deployment. IMHO the dlls will be copied to the output folder and the Nuget Package will include them, the only side effect I can see is increasing the project deployment size, but I can fix this changing the Behavior from a `Blend Behavior` to an `Attached Property Behavior`.